.appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   global:
-    LIBRDKAFKA_NUGET_VERSION: 1.3.0
+    LIBRDKAFKA_NUGET_VERSION: 1.4.0-PRE6
```

Comment:
Let me create an RC off master, this one is quite dated now.

Reply:
alternatively I can just point it at master until we are ready for a release

Reply:
There is no master nuget package though.